fbuapp/Views/PostTableView.m

```diff
@@ -97,12 +97,18 @@ -(void)locationManager:(CLLocationManager *)manager didFailWithError:(NSError *)
     NSLog(@"Error: %@",error.description);
 }
 
+
 - (UITableViewCell *)tableView:(UITableView *)tableView
          cellForRowAtIndexPath:(NSIndexPath *)indexPath
 {
     PostCell *cell = [tableView dequeueReusableCellWithIdentifier:kTableViewPostCell];
     cell.delegate = self;
     Post *post = self.posts[indexPath.row];
+    
+
+    
+//    cell.eventAuthor.text = [PFQuery getUserObjectWithId: post[@"eventAuthor"]][@"firstName"];
```

Comment:
probably delete this when merging with camryn's branch.

Reply:
ok